Core/src/Geometry/CuboidVolumeBuilder.cpp

```diff
@@ -152,30 +152,6 @@ std::shared_ptr<Acts::TrackingVolume> Acts::CuboidVolumeBuilder::buildVolume(
   auto bounds = std::make_shared<const CuboidVolumeBounds>(
       cfg.length.x() * 0.5, cfg.length.y() * 0.5, cfg.length.z() * 0.5);
 
-  if (cfg.layerCfg.empty()) {
```

Comment:
Should this be an error in that case?

Reply:
to me it seems fine to have a volume without layers. in the unit tests I saw this was used to have dense material in a box basically